packages/block-editor/src/components/inserter/menu.js

```diff
@@ -30,6 +30,7 @@ import {
 } from '@wordpress/components';
 import {
 	getCategories,
+	getCollections,
```

Comment:
Wouldn't it be more reliable to get both collections and categories through `withSelect` HOC?